index.html

```diff
@@ -1,47 +1,234 @@
-  <div class="main-text">
-    <span>Hey there.</span>
-    <span>I'm Aly.</span>
+  </header>
+  <section class="main">
+    <div class="background-main absolute"></div>
+    <div class="main-text">
+      <span>Hey there.</span>
+      <span>I'm Aly.</span>
       <h2 class="colored-text"><span>I'm a software </span>
-        <span>Developer</span></h2>
-  </div>
-  <div class="long-text">
-      <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-          don’t hesitate to contact me.</p>
-  </div>
-  <ul class="social-icons">
-    <li><a rel="noopener noreferrer" href="mailto:alyatalla93@gmail.com" target="_blank"><i class="fa-solid fa-envelope"></i></a></li>
-    <li><a rel="noopener noreferrer" href="https://github.com/AlyAtalla" target="_blank"><i class="fa-brands fa-github"></i></a></li>
-    <li><a rel="noopener noreferrer" href="https://twitter.com/aliatalla93" target="_blank"><i class="fa-brands fa-square-twitter"></i></a></li>
-    <li><a rel="noopener noreferrer" href="https://www.linkedin.com/in/aly-atalla-8b83a0231/" target="_blank"><i class="fa-brands fa-linkedin"></i></a></li>
-</ul>
-<a class="scroll" href="#works">
-    <i class="fa-solid fa-chevron-down"></i>
-</a>
+        <span>Developer</span>
+      </h2>
+    </div>
+    <div class="long-text">
+      <p> I can help you build a product , feature or website Look through some of my work and experience! If you like
+        what you see and have a project you need coded,
+        don’t hesitate to contact me.</p>
+    </div>
+    <ul class="social-icons">
+      <li><a rel="noopener noreferrer" href="mailto:alyatalla93@gmail.com" target="_blank"><i
+            class="fa-solid fa-envelope"></i></a></li>
+      <li><a rel="noopener noreferrer" href="https://github.com/AlyAtalla" target="_blank"><i
+            class="fa-brands fa-github"></i></a></li>
+      <li><a rel="noopener noreferrer" href="https://twitter.com/aliatalla93" target="_blank"><i
+            class="fa-brands fa-square-twitter"></i></a></li>
+      <li><a rel="noopener noreferrer" href="https://www.linkedin.com/in/aly-atalla-8b83a0231/" target="_blank"><i
+            class="fa-brands fa-linkedin"></i></a></li>
+    </ul>
+    <a class="scroll" href="#works">
+      <i class="fa-solid fa-chevron-down"></i>
+    </a>
+  </section>
+  <section class="works" id="works">
+    <div class="main-works">
+      <span>My Recent</span><span> Works</span>
+      <img src="images/Indicator.png" alt="line Image" class="line-image">
+    </div>
+    <div class="breaker"></div>
+    <div class="cards"></div>
+    <div class="card">
+      <div class="card-img">
+        <img src="images/project1.jpeg" alt="Project Image">
+      </div>
+      <div class="card-body">
+        <span>A website for</span><span> English language school</span>
+        <ul class="language-blocks">
```

Comment:
- [ ] Good work up to this point but there are aspects you can till improve on: 
  - Your project cards are aligned to the left. Lindly center them to match the original design.
  
![image](https://github.com/AlyAtalla/AlyAtalla/assets/98453979/1a09b010-9765-4763-93ee-2a818f74b17c)

- [ ] The same case also applies in your about me section
  - Kindly also adjust the font used on the title as it appears to be different from the original design
  
![image](https://github.com/AlyAtalla/AlyAtalla/assets/98453979/fec93fdb-3315-4a20-9797-3d31496523a1)

- [ ] Kindly note that you can edit the pull request and write the changes introduced without having to write a new comment. Click on the three-dotted button and you will see an edit option

![image](https://github.com/AlyAtalla/AlyAtalla/assets/98453979/5ad697e7-dce9-4079-b504-4bdf60201354)